website/static/css/quick-project-search-plugin.css

```diff
@@ -0,0 +1,45 @@
+
+.node-styling {
+    background-color: #fcfcfc;
+    border: 1px solid #cccccc;
+}
+
+button {
```

Comment:
In general it's best to not use very generic identifiers to define rules in CSS. Other aspects of this page will use button. Same with using class names like 'selected' which may be very common.You can either use specific selectors like ```.quick-project button```  or namespace individual elements like ```.qps-button``` I prefer the first method so that you don't add classes to all the elements. The trick is to use a top level unique identifier and use specific classes for things underneath.